apps/docs/content/guides/functions/quickstart.mdx

```diff
@@ -1,126 +1,155 @@
 
-```js name=JavaScript
-import { createClient } from '@supabase/supabase-js'
+1. Navigate to your Edge Functions > Secrets page.
+2. Look for the option to add environment variables.
+3. Add a new environment variable with the key `OPENAI_API_KEY` and set its value to your actual OpenAI API key.
 
-const supabase = createClient(process.env.SUPABASE_URL, process.env.SUPABASE_ANON_KEY)
+Once you've set this environment variable, your edge functions will be able to access the OPENAI_API_KEY securely without hardcoding it into the function code. This is a best practice for keeping sensitive information safe.
 
-const { data, error } = await supabase.functions.invoke('hello-world', {
-  body: { name: 'Functions' },
-})
-```
+With your variable set, you can test by sending a request via the dashboard. Navigate to the function's details page, and click on the test button. Then provide a Request Body your function expects.
+
+<div style={{ width: '60%', position: 'relative', aspectRatio: 'auto' }}>
+  <Image
+    alt="Provide a request body to test your function"
+    src={{
+      light: '/docs/img/guides/functions/dashboard/test-request-body--light.png',
+      dark: '/docs/img/guides/functions/dashboard/test-request-body--light.png',
+    }}
+    zoomable
+  />
+</div>
 
-</$CodeTabs>
+## Editing functions from the Dashboard
 
-<Admonition type="tip" label="Where is my SUPABASE_ANON_KEY?">
+<Admonition type="caution" label="Be careful: there is currently no version control for edits">
 
-Run `supabase status` to see your local credentials.
+The Dashboard's Edge Function editor currently does not support versioning or rollbacks. We recommend using it only for quick testing and prototypes. When you’re ready to go to production, store Edge Functions code in a source code repository (e.g., git) and deploy it using one of the [CI integrations](https://supabase.com/docs/guides/functions/cicd-workflow).
 
 </Admonition>
 
-You should see the response `{ "message":"Hello Functions!" }`.
+1. From the functions page, click on the function you want to edit. From the function page, click on the Code tab.
 
-If you execute the function with a different payload, the response will change.
+2. This opens up a code editor in the dashboard where you can see your deployed function's code.
 
-Modify the `--data '{"name":"Functions"}'` line to `--data '{"name":"World"}'` and try invoking the command again.
+3. Modify the code as needed, then click Deploy updates. This will overwrite the existing deployment with the newly edited function code.
 
 ## Next steps
 
-Check out the [Deploy to Production](/docs/guides/functions/deploy) guide to make your Edge Function available to the world.
+Check out the [Local development](/docs/guides/functions/local-quickstart) guide for more details on working with Edge Functions.
 
 Read on for some common development tips.
```

Comment:
Shall we move the development tips to a separate page? That makes Quickstart look succinct. Also, currently they are repeated on both Quickstart and Local Development. Better to keep them in a single and link to from other pages.